aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -369,7 +369,7 @@ Scaffolding generated the following highlighted code in `Program.cs`:
 
 # [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
 
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie80/Program.cs?name=snippet_FirstSQLite&highlight=3-4)]
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie80/Program.cs?name=snippet_FirstSQLServer&highlight=3-4)]
```

Comment:
```suggestion
[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie80/Program.cs?name=snippet_FirstSQLite&highlight=3-4)]
```

Reply:
The link should be to example code for registering a SQLite database context using DI rather than SQL Server for the Visual Studio Code version (tab) of this document.  Changing back to SQLite and generating a build to see why that code is not being picked up in the doc build.